src/Schema/Extensions/GraphQLExtension.php

```diff
@@ -14,7 +14,7 @@
      *
      * @return DocumentAST
      */
-    public function manipulateSchema(DocumentAST $current, DocumentAST $original)
+    public function manipulateSchema(DocumentAST $current, DocumentAST $original): DocumentAST
```

Comment:
Until this becomes the default way of doing things in Laravel, let's not force the user to add return types on classes they will be extending.

Reply:
Agreed when we are dealing with stuff in Laravel-Land where we are not responsible for dealing with the result (as in the Controller). In this case, we handle the return value and always, positively and strictly need a `DocumentAST` so i would advocate towards keeping it there.